browser/lib/util/defaults.js

```diff
@@ -15,11 +15,22 @@ var Defaults = {
 	transports:               ['web_socket', 'xhr', 'iframe', 'jsonp']
 };
 
+/* If an environment option is provided, the environment is prefixed to the domain
+   i.e. rest.ably.io with environment sandbox becomes sandbox-rest.ably.io */
+Defaults.environmentHost = function(environment, host) {
+	if (!environment || (String(environment).toLowerCase() === 'production')) {
```

Comment:
You'll get a warning when the client library fails to connect to `true-rest.ably.io`